README.md

```diff
@@ -233,51 +218,9 @@ We recommend creating JavaScript-specific classes to bind to, prefixed with `.js
     }
     ```
 
-### Mixins
```

Comment:
Yeah, I removed it because it only recommended against using argumentless mixins in favor of extending placeholders. I'll push a new revision that improves this section.